docs/management/admin/trusted-apps.asciidoc

```diff
@@ -2,11 +2,13 @@
 [chapter, role="xpack"]
 = Trusted applications
 
-You can add Windows, macOS, and Linux applications that should be trusted. By adding these trusted applications, you can use {elastic-sec} without compatibility or performance issues with other installed applications on your system. Trusted applications apply only to hosts running {elastic-defend}.
+You can add Windows, macOS, and Linux applications that should be trusted, such as other antivirus or endpoint security applications. Trusted applications are designed to help mitigate performance issues and incompatibilities with other endpoint software installed on your hosts. Trusted applications apply only to hosts running the {elastic-defend} integration.
 
 NOTE: You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].
 
-Trusted applications are designed to help mitigate performance issues and incompatibilities with other endpoint software. However, they create blindspots for {elastic-sec}. One avenue attackers use to exploit these blindspots is by DLL (Dynamic Link Library) side-loading, where they leverage processes signed by trusted vendors -- such as antivirus software -- to execute their malicious DLLs. Such activity appears to originate from the trusted vendor's process.
+Trusted applications create blindspots for {elastic-defend}, because the applications are no longer monitored for threats. One avenue attackers use to exploit these blindspots is by DLL (Dynamic Link Library) side-loading, where they leverage processes signed by trusted vendors -- such as antivirus software -- to execute their malicious DLLs. Such activity appears to originate from the trusted vendor's process.
+
+Trusted applications may still generate alerts in some cases, such as if the application's process events indicate malicious behavior. To reduce false positive alerts, add an <<endpoint-rule-exceptions,Endpoint alert exception>>, which prevents {elastic-defend} from generating alerts. To compare trusted applications with other endpoint artifacts, refer to <<endpoint-artifacts>>.
```

Comment:
Same confusion for me here as on the Endpoint artifacts page. I thought it just said trusted applications are not monitored for threats, but now it's saying they can still generate alerts if they're doing malicious behavior. Maybe I am not understanding the distinction between being monitored for threats and being monitored for malicious behavior? I hope I'm not the only person confused by this.

Reply:
My response above: https://github.com/elastic/security-docs/pull/2516/files#r992737049